src/core/lib/load_balancing/lb_policy.cc

```diff
@@ -18,12 +18,11 @@
 
 #include <grpc/support/port_platform.h>
 
-#include "src/core/lib/load_balancing/lb_policy.h"
+#include "lb_policy.h"
```

Comment:
Looks like iwyu did something weird here.  I think we still need the full path to this include, but it seems to have changed the path here and then added the right path as a separate include on line 25.

Reply:
I haven't seen that one before. Fixed.